src/functions/functions-api-client-internal.ts

```diff
@@ -54,6 +55,61 @@ export class FunctionsApiClient {
     }
     this.httpClient = new AuthorizedHttpClient(app as FirebaseApp);
   }
+  /**
+   * Deletes a task from a queue.
+   *
+   * @param id - The ID of the task to delete.
+   * @param functionName - The functionName of the queue.
```

Comment:
Is functionName a literal?

But it probably makes more sense to just say "function name" in the definition/description.